weblate/api/serializers.py

```diff
@@ -502,6 +503,62 @@ class Meta:
         )
 
 
+class TermSerializer(serializers.ModelSerializer):
+    language = LanguageSerializer(required=False)
+
+    class Meta:
+        model = Term
+        fields = (
+            "language",
+            "id",
+            "source",
+            "target",
+        )
+
+    def to_internal_value(self, data):
+        result = super().to_internal_value(data)
+        if "language" in result:
+            result["language"] = Language.objects.get(
+                code=result["language"].get("code")
```

Comment:
The `get` invocation has no benefits here:

```suggestion
                code=result["language"]["code"]
```